CONTRIBUTING.md

```diff
@@ -1,7 +1,7 @@
 # Contributing Guide
 
 ## How to Contribute
-1. Find a [issue tagged with "help wanted"](https://github.com/TheOdinProject/theodinproject/labels/Help%20Wanted) to work on. Please comment on the issue to let others know you are working on it.
+1. Find a [issue tagged with "help wanted"](https://github.com/TheOdinProject/theodinproject/labels/Help%20Wanted) to work on. Depending on the issue, you can choose to work on one or more acceptance criteria. Please comment on the issue stating what you'd like to work on and wait to be assigned before working on it. If you would like to propose a change that is not covered in an open issue, you can create a new issue. **If the change is fairly small, such as fixing a typo or updating a link, you can submit a PR to propose the change.**
```

Comment:
Can you just clarify on the situations where they could work on one or more acceptance criteria. Usually an issue is a small refined problem maybe from a larger feature where we would only merge once all the acceptance criteria are met.